src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -527,9 +527,6 @@ def __generate_test_lists(self, test_dict_cache: dict, test_dict_runtime: dict):
 
     def __prepare_smart_filters(self, proved_test_dict: dict):
         def_length = len(self._command) + len(" --gtest_filter=")
-        if constants.IS_WIN:
-            # subprocess add cmd.exe to the command line on Windows if shell=True
-            def_length += len(f'{os.environ.get("COMSPEC", "cmd.exe")} /C ')
```

Comment:
Honestly, I did not get the change idea... Could you please share technical details?

Reply:
Please check the conformance run locally to avoid breaking od the script